dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -242,6 +247,14 @@ public class ConnectorArguments extends DefaultArguments {
           .ofType(Integer.class)
           .describedAs("N");
 
+  private final OptionSpec<ChronoUnit> optionQueryLogInterval =
+      parser
+          .accepts(OPT_QUERY_LOG_INTERVAL, "The interval between each query log dump")
+          .withOptionalArg()
+          .ofType(ChronoUnit.class)
```

Comment:
If `ChronoUnit` is printed out in the `--help` output, then I would suggest to change this type to `String` or `Duration`. Also, I think we should replace printing Java types with something more user-friendly for all the options in the long run.

(I don't expect any changes in `ZonedIntervalIterable`. I still support converting to `ChronoUnit` internally, possibly in `getQueryLogInterval` method or somewhere else, where it is not visible in the help output.)